site/docs/guidelines/accessibility.mdx

```diff
@@ -29,7 +29,7 @@ In addition to WCAG 2.1 and 2.0, HDS also refers to the following guidelines:
 - <Link href="https://dev.hel.fi/accessibility" external>Develop with Helsinki - accessibility guidelines for developers</Link>.
 
 ## HDS accessibility process
-**After HDS beta release, new components and patterns are not released before their accessibility has been audited by a third party auditor.** If components or patterns are released without an audit, it will be clearly indicated in the documentation. This may happen if the component is quickly needed by some project or if it takes considerably long time to make the component accessible.
+**New components and patterns are not released before their accessibility has been audited by a third party auditor. This includes components marked as Drafts.** If components or patterns are released without an audit, it will be clearly indicated in the documentation. This may happen if the component is quickly needed by some project or if it takes considerably long time to make the component accessible.
```

Comment:
> **New components and patterns are not released before their accessibility has been audited by a third party auditor. This includes components marked as Drafts.** If components or patterns are released without an audit, it will be clearly indicated in the documentation. This may happen if the component is quickly needed by some project or if it takes considerably long time to make the component accessible.

**New components and patterns are not released before their accessibility has been audited by a third-party auditor. This includes components marked as Drafts.** If components or patterns are released without an audit, it will be clearly indicated in the documentation. This may happen if the component is quickly needed by some project or if it takes a considerably long time to make the component accessible.